addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRule.java

```diff
@@ -322,6 +323,42 @@ private String readPayloadsFile() {
         return "";
     }
 
+    private boolean doesNotMatch(String responseBody, List<String> notContentStrings) {
```

Comment:
- Can be static (same for `doesMatch` and `doesBinRegexMatch`).
 - This is `doesMatch` negated (though we do need to check if empty beforehand), we could call that instead of duplicating.